localisation/00_wc_job_titles.csv

```diff
@@ -218,22 +218,6 @@ job_treasurer_murloc_group;Mmmml Mmgr;Mmmml Mmgr;Mmmml Mmgr;;Mmmml Mmgr;;;;;;;;;
 job_treasurer_female_murloc_group;Mmmml Mmgr;Mmmml Mmgr;Mmmml Mmgr;;Mmmml Mmgr;;;;;;;;;x
 job_spymaster_murloc_group;Furl Mmmurlok;Furl Mmmurlok;Furl Mmmurlok;;Furl Mmmurlok;;;;;;;;;x
 job_spymaster_female_murloc_group;Furl Mmmurlok;Furl Mmmurlok;Furl Mmmurlok;;Furl Mmmurlok;;;;;;;;;x
-job_chancellor_naga_group;Herald;;Vorbote;;;;;;;;;;;x
-job_chancellor_female_naga_group;Herald;;Vorbotin;;;;;;;;;;;x
-job_marshal_naga_group;General;;General;;;;;;;;;;;x
-job_marshal_female_naga_group;General;;Gener�lin;;;;;;;;;;;x
-job_treasurer_naga_group;Tidemaster;;Gezeitenmeister;;;;;;;;;;;x
-job_treasurer_female_naga_group;Tidemistress;;Gezeitenmeisterin;;;;;;;;;;;x
-job_spymaster_naga_group;Triton;;Triton;;;;;;;;;;;x
-job_spymaster_female_naga_group;Siren;;Sirene;;;;;;;;;;;x
-job_chancellor_night_elf_group;Emissary;;Botschafter;;;;;;;;;;;x
-job_chancellor_female_night_elf_group;Emissary;;Botschafterin;;;;;;;;;;;x
-job_marshal_night_elf_group;General;G�n�ral;General;;;;;;;;;;;x
-job_marshal_female_night_elf_group;General;G�n�ral;Gener�lin;;;;;;;;;;;x
-job_treasurer_night_elf_group;Quartermaster;;Quartiermeister;;;;;;;;;;;x
-job_treasurer_female_night_elf_group;Quartermaster;;Quartiermeisterin;;;;;;;;;;;x
-job_spymaster_night_elf_group;Nightblade;;Nachtklinge;;;;;;;;;;;x
-job_spymaster_female_night_elf_group;Nightblade;;Nachtklinge;;;;;;;;;;;x
```

Comment:
Wait, what happened to these titles?

Reply:
They're now work only on culture, not culture group, I don't think I removed them entirely, jsut Github being kinda retarded

Reply:
Nope, definitely deleted